app/code/Magento/ProductVideo/view/adminhtml/web/js/get-video-information.js

```diff
@@ -559,7 +559,7 @@ define([
                 this._videoInformation = null;
                 this.element.val('');
                 alert({
-                    content: 'Error: "' + data + '"'
+                    content: $.mage.__('Error: ') + '"' + data + '"'
```

Comment:
It would be better to return the whole translated error message from the backend instead of concatenation

Reply:
Hello @sidolov , in this function, "data" is translated before pass to the `_onVideoInvalid` function. So if we translate whole message, it will translate double times ("data")

Reply:
I ment remove translation call from frontend at all and move `Error:` to the backend